mono/metadata/w32handle.c

```diff
@@ -1284,13 +1284,6 @@ mono_w32handle_wait_multiple (gpointer *handles, gsize nhandles, gboolean waital
 	if (timeout != MONO_INFINITE_WAIT)
 		start = mono_msec_ticks ();
 
-	for (i = 0; i < nhandles; ++i) {
```

Comment:
(This seemed a lot more likely before I learned that SafeHandle/WaitHandle protects from disposing during inflight operations, but I don't know what that mechanism entails. It may at least mean that the buggy _unref is coming from runtime code)